### ISSUE TITLE
wine support

### ISSUE THREAD

**Issue by max630** [git-for-windows/git#150]
I cannot install the sdk under wine. `pacman` hangs at startup and does nothing. Is wine or other build from linux in planned to be supported? If not it's quite disappointing change compared to older msys

**Comment by elieux** [comment]
Git for Windows SDK is based on MSYS2 and MSYS2-on-Wine (and Cygwin-on-Wine in general) is largely unexplored territory. There's at least one guy testing these scenarios and making patches for Wine, but it's expected to take some time before it's stable enough for day-to-day work.

**Comment by max630** [comment]
There could be at least standalone installer with all packages downloaded and installed. I would expect the actual git build functionality to work.

**Comment by dscho** [comment]
@max630 the primary purpose of the Git SDK is to support contributors for the Git for Windows project. The target audience consists therefore of Windows developers working on Windows.

We did have one "full Git SDK" installer that weighed in with almost 200MB, which I find an unreasonable maintenance burden, speaking for myself. Unless you fix Wine to cope with Pacman, you would not even be able to update the packages as required to stay up-to-date.

In short: if you want to see this work correctly, please invest the time and effort to make it so.

**Comment by dscho** [comment]
I do not get the impression that this ticket is going anywhere. Therefore I am closing it.

**Comment by mkllnk** [comment]
Since the wine project would like to run every program as Windows would do, I created an issue at their bug tracker: https://bugs.winehq.org/show_bug.cgi?id=39634

**Comment by max630** [comment]
I'm sure the real place to fix is somewhere in msys2 runtime. But I don't see any realistic option to debug it, so I gave up.

**Comment by fracting** [comment]
Please try Wine Staging, we've spend a lot of time to make MSYS2 works on Wine Staging, if there is any developer using Wine to test / developer Win32 open source project that would be great interesting.

https://github.com/fracting/wine-fracting/wiki/MSYS2-on-Wine#build-status

**Comment by mkllnk** [comment]
Yes, works for me. I downloaded 1.7.55-staging via PlayOnLinux but then started it on the command line:
```bash
export PATH="$HOME/.PlayOnLinux/wine/linux-x86/1.7.55-staging/bin:$PATH"
export WINEPREFIX="/tmp/1.7.55-staging"
mkdir -p "$WINEPREFIX"
wine ~/Downloads/Git-2.6.3-32-bit.exe
# just click next all the time
cd /tmp/1.7.55-staging/drive_c/Program\ Files/Git/
wine wineconsole bin/bash --login -i
# got a shell and can invoke git without error messages
```

I'm working on a 64bit Debian Wheezy system with some 32bit libs installed for wine.

Thank you people, that's awesome!

**Comment by max630** [comment]
I meant sdk, not the git itself (it does not use the msys libraries afaik). Though if you are able to run bash it a good sign.

**Comment by fracting** [comment]
@mkllnk Cool, if you found any bug please report to Wine Staging and CC me.

@max630 the Git for Windows SDK support is also in my todo list, I haven't have time to test yet, but if you find any bug please also CC me.

Thanks!
